bazel/repos.bzl

```diff
@@ -9,14 +9,20 @@
 # instructions at https://github.com/3rdparty/eventuals-grpc.
 ########################################################################
 
-load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
 load("//3rdparty/eventuals:repos.bzl", eventuals_repos = "repos")
+load("//3rdparty/pyprotoc-plugin:repos.bzl", pyprotoc_plugin_repos = "repos")
 load("//3rdparty/stout-borrowed-ptr:repos.bzl", stout_borrowed_ptr_repos = "repos")
 load("//3rdparty/stout-notification:repos.bzl", stout_notification_repos = "repos")
-load("//3rdparty/pyprotoc-plugin:repos.bzl", pyprotoc_plugin_repos = "repos")
 
 def repos(external = True, repo_mapping = {}):
+    """Adding eventuals, borrowed_ptr, notification & plugin repos (3rdparty).
```

Comment:
Rather than having to update the comment every time let's just keep it simple and say "Adds repositories we depend on."